src/Persistance/RepositoryImplementations/SystemRegisterRepository.cs

```diff
@@ -77,7 +82,9 @@ public async Task<List<RegisteredSystem>> GetAllActiveSystems()
 
             command.Parameters.AddWithValue("registered_system_id", toBeInserted.SystemTypeId);
             command.Parameters.AddWithValue("system_vendor", toBeInserted.SystemVendor);
-            command.Parameters.AddWithValue("description", toBeInserted.Description);
+            command.Parameters.AddWithValue("description", toBeInserted.FriendlyProductName);
```

Comment:
is this parameter name right, i remember seeing friendlyproductname

Reply:
yes, it is